[Cookbook] [Deployment] Added note about Nginx

phansys opened the pull request:
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3+ |
| Fixed tickets |  |

xabbuh commented on cookbook/deployment/heroku.rst:
```diff
@@ -106,6 +106,15 @@ directory of the application and add just the following content:
 
+.. note::
+
+    Nginx is also available in Heroku. If you prefer to use it, you can create a
```
on Heroku

phansys replied:
Thank you @xabbuh, fixed.

xabbuh commented on cookbook/deployment/heroku.rst:
```diff
@@ -106,6 +106,15 @@ directory of the application and add just the following content:
+.. note::
+
+    Nginx is also available on Heroku. If you prefer to use it, you can create a
+    Nginx config file (``nginx_app.conf`` for instance) to point from Procfile as described in `Heroku docs`_:
```
I think we should reword the paragraph a bit like this:

> If you prefer to use Nginx, which is also available on Heroku, you can create a configuration file for it and point to it from your Procfile as described in the Heroku documentation.

What do you think @phansys?

phansys replied:
This sounds better. Updated.

xabbuh commented:
:+1: nice work @phansys

phansys commented:
Should I rebase the PR and resolve the conflicts?

phansys committed:
[Cookbook] [Deployment] Added note about Nginx setup

| Q             | A
| ------------- | ---
| Doc fix?      | yes
| New docs?     | no
| Applies to    | 2.3+
| Fixed tickets |

xabbuh commented:
@phansys That would be nice.

phansys commented:
@xabbuh, done. Thanks.

weaverryan commented:
I love it! Thanks Javier!
